README.md

```diff
@@ -79,10 +99,10 @@ You can also send byte array directly:
 client.Send([]string{"another client address"}, []byte{1, 2, 3, 4, 5}, 0)
 ```
 
-Or publish text message to a topic (subscribe is done through [nkn-wallet-js](https://github.com/nknorg/nkn-wallet-js)):
+Or publish text message to a topic:
 
 ```go
-client.Publish("topic", []byte("hello world!"), 0)
+client.Publish("topic", 0, []byte("hello world!"), 0)
```

Comment:
Some simple clarification may be needed to explain what these two 0 means...

Reply:
done